lib/v2/dockerhub/tag.js

```diff
@@ -0,0 +1,30 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const { owner, image } = ctx.params;
+
+    const namespace = `${owner}/${image}`;
+
+    const link = `https://hub.docker.com/r/${namespace}`;
+
+    const data = await got.get(`https://hub.docker.com/v2/repositories/${namespace}/tags/`);
+    const metadata = await got.get(`https://hub.docker.com/v2/repositories/${namespace}/`);
+
+    const tags = data.data.results;
+
+    ctx.state.data = {
+        title: `${namespace} tags`,
+        description: metadata.data.description,
+        link,
+        language: 'en',
+        item: tags.map((item) => ({
+            title: `${namespace}:${item.name} was updated`,
+            description: `${namespace}:${item.name} was updated, supporting the architectures of ${item.images.map((img) => `${img.os}/${img.architecture}`).join(', ')}`,
+            link: `https://hub.docker.com/layers/${owner === 'library' ? `${image}/` : ''}${namespace}/${item.name}/images/${item.images[0].digest.replace(':', '-')}`,
+            author: owner,
+            pubDate: parseDate(item.tag_last_pushed),
+            guid: item.tag_last_pushed,
```

Comment:
Consider not setting the guid.
If guid is not set, it will use `link` instead, which has a SHA256 that is more collision resistance than a timestamp imo.

Reply:
Give nthat `link` is just the first architecture variant of the image:
* If the image didn't change but the order of the architecture changed, we might get different SHA256. Thus, the same tag push might give different hashes.
* If the image on a certain architecture changed but not the one we pick SHA256 on, Thus, a change might not be detected.

Probably we can collect all SHA256s and integrate them to a single GUID for the sake of robustness.